frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -0,0 +1,294 @@
+import { CheckCircle2, Code, PlusCircle, RefreshCw } from "lucide-react";
+import React from "react";
+import ExternalLink from "src/components/ExternalLink";
+import { AlertDialog, AlertDialogCancel, AlertDialogContent, AlertDialogDescription, AlertDialogFooter, AlertDialogHeader, AlertDialogTitle, AlertDialogTrigger } from "src/components/ui/alert-dialog";
+import { Button } from "src/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+import { Input } from "src/components/ui/input";
+import { Label } from "src/components/ui/label";
+import { LoadingButton } from "src/components/ui/loading-button";
+import { useToast } from "src/components/ui/use-toast";
+import { useApps } from "src/hooks/useApps";
+import { useTransactions } from "src/hooks/useTransactions";
+import { createApp } from "src/requests/createApp";
+import { CreateAppRequest, UpdateAppRequest } from "src/types";
+import { handleRequestError } from "src/utils/handleRequestError";
+import { request } from "src/utils/request";
+
+const appName = `ZapPlanner - Alby Hub`;
+const recipientLightningAddress = "hub@getalby.com";
+
+function SupportAlby() {
+  const { data: apps, mutate: reloadApps } = useApps();
+  const subscription = apps?.find(x => x.name === appName);
+  const { toast } = useToast();
+  const { data: transactions, isLoading } = useTransactions(subscription?.id);
+
+  const [amount, setAmount] = React.useState("");
+  const [senderName, setSenderName] = React.useState("");
+  const [isSubmitting, setSubmitting] = React.useState(false);
+  const [open, setOpen] = React.useState(false);
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setSubmitting(true);
+    try {
+      if (apps?.some((existingApp) => existingApp.name === appName)) {
+        throw new Error("A connection with the same name already exists.");
+      }
+
+      const parsedAmount = parseInt(amount);
+      if (isNaN(parsedAmount) || parsedAmount < 1) {
+        throw new Error("Invalid amount");
+      }
```

Comment:
Should we also ensure the payment is at least 1000 sats, otherwise is it really worthwhile?